assets/js/blocks/cart-checkout/payment-methods/test/payment-methods.js

```diff
@@ -14,11 +11,20 @@ import {
 	usePaymentMethodDataContext,
 } from '@woocommerce/base-context';
 
+/**
+ * Internal dependencies
+ */
+import * as useStoreCartHook from '../../../../base/context/hooks/cart/use-store-cart';
+
+// Somewhere in your test case or test suite
+useStoreCartHook.useStoreCart = jest
+	.fn()
+	.mockReturnValue( useStoreCartHook.defaultCartData );
+
```

Comment:
Can we still know if this was called or not? I'm asking because I wanted to copy the code to here but we really need to know if `wc/store/cart` was called more than once, which would be a breaking test.

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/6da64165025e7a2afc1782e4b278d72536e7b754/assets/js/blocks/cart-checkout/cart/test/block.js#L54

Is it important here to know if `wc/store/cart` was called more than once?

Reply:
To test if this mock was called this should work.
```
	expect( useStoreCartHook.useStoreCart ).toHaveBeenCalledTimes( x );
```

> Can we still know if this was called or not?

Do you mean in the scope of this test? Because I don't think this is something this test should check. After adding cart as part of the argument this test requests `wc/store/cart` and because it was not mocked it was failing. So I just mocked useStoreCart to not make the fetch call and just return a cart object.

For the block test, you actually need to mock the fetch utility to test that an actual call was made. So I would not copy that there as well.